packages/ftp/test/write.test.js

```diff
@@ -0,0 +1,28 @@
+import { strictEqual } from 'assert'
+import fs from 'fs'
+import { dirname, resolve } from 'path'
+import getStream from 'get-stream'
+import write from '../write.js'
+import { withServer } from './support/server.js'
+import { ftpConfigurations } from './support/serverConfigurations.js'
+
+export const __dirname = dirname(new URL(import.meta.url).pathname)
+
+describe('write', () => {
+  ftpConfigurations.forEach(
+    ([label, serverFactory, additionalOptions]) => {
+      it(`writes file at the given path ${label}`, async () => {
+        await withServer(serverFactory, async server => {
+          const options = { ...server.options, ...additionalOptions }
+
+          const stream = await write({ filename: 'tmp/foo.txt', ...options })
+          const input = fs.createReadStream(resolve(__dirname, 'support/data/xyz.txt'))
+          input.pipe(stream)
+
+          await getStream(stream)
```

Comment:
@tpluscode I will let you check the right thing to use there, as it's the thing that is in trouble. Maybe `getStreamAsArray`? Or any other method exported by `get-stream`.

Reply:
The error I get is because the `stream` is not readable but writable